README.md

```diff
@@ -655,38 +746,63 @@
     });
     ```
 
-  - Place 1 space before anonymous function parenthesis.
+  - Place 1 space before the opening parenthesis in control statements (`if`, `while` etc.). Place no space before the argument list in function calls and declarations.
 
     ```javascript
-	// bad
```

Comment:
If you think about ES6 method shorthand, then in my mind `fightIf (isJedi) { }` makes it much clearer that we are looking at a function definition. Does that help?